docker-compose.yml

```diff
@@ -9,10 +11,10 @@ services:
       - "4567:4567"
     depends_on:
       - validator_service
-  cqf_ruler:
-    image: mitrehealthdocker/cqf-ruler-preloaded:0.6.0.no-vs
-    ports:
-      - "8080:8080"
+  # cqf_ruler:
```

Comment:
Was cqf-ruler commented out for testing or do we want to keep it commented out for when this is used for testing at the Connectathon?